Commands/Public/messages.js

```diff
@@ -0,0 +1,81 @@
+const moment = require("moment");
+
+module.exports = async ({ client, Constants: { Colors } }, { serverDocument, memberDocument }, msg, commandData) => {
+	if (msg.suffix === "me") {
+		return msg.send({
+			embed: {
+				author: {
+					name: client.getName(serverDocument, msg.author),
+					iconURL: msg.author.displayAvatarURL({ size: 32 }),
```

Comment:
Actually for the client the size does matter since it downloads the full-size image even if it's only displayed at 20x20px. A very minor thing but if it's that easy to fix... ¯\\\_(ツ)_/¯

Reply:
Then its fine:D